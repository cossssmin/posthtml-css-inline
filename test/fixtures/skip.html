<!DOCTYPE html>
<html>
<head>
  <style no-inline>
    .flex {
      display: flex;
    }

    p {
      color: red;
    }
  </style>
  <style>
    .blue {
      color: blue;
    }
    .inline {
      display: inline;
    }
    #title {
      color: #000;
    }
  </style>
</head>
<body>
  <p class="flex blue">a</p>
  <div class="blue inline" no-inline>b</div>
  <h1 id="title" no-inline>Title</h1>
</body>
</html>
