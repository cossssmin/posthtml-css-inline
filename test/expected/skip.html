<!DOCTYPE html>
<html>
<head>
  <style>
    .flex {
      display: flex;
    }

    p {
      color: red;
    }
  </style>
  <style>#title {
      color: #000;
    } .blue {
      color: blue;
    } .inline {
      display: inline;
    }</style>
</head>
<body>
  <p class="flex blue" style="color: blue">a</p>
  <div class="blue inline">b</div>
  <h1 id="title">Title</h1>
</body>
</html>
